flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamGraph.java

```diff
@@ -692,69 +762,15 @@ void setTransformationUserHash(Integer nodeId, String nodeHash) {
 		}
 	}
 
-	public StreamNode getStreamNode(Integer vertexID) {
-		return streamNodes.get(vertexID);
-	}
-
-	protected Collection<? extends Integer> getVertexIDs() {
+	Collection<? extends Integer> getVertexIDs() {
 		return streamNodes.keySet();
 	}
 
-	@VisibleForTesting
-	public List<StreamEdge> getStreamEdges(int sourceId) {
-		return getStreamNode(sourceId).getOutEdges();
-	}
-
-	@VisibleForTesting
-	public List<StreamEdge> getStreamEdges(int sourceId, int targetId) {
-		List<StreamEdge> result = new ArrayList<>();
-		for (StreamEdge edge : getStreamNode(sourceId).getOutEdges()) {
-			if (edge.getTargetId() == targetId) {
-				result.add(edge);
-			}
-		}
-		return result;
-	}
-
-	@VisibleForTesting
-	@Deprecated
-	public List<StreamEdge> getStreamEdgesOrThrow(int sourceId, int targetId) {
-		List<StreamEdge> result = getStreamEdges(sourceId, targetId);
-		if (result.isEmpty()) {
-			throw new RuntimeException("No such edge in stream graph: " + sourceId + " -> " + targetId);
-		}
-		return result;
-	}
-
-	public Collection<Integer> getSourceIDs() {
-		return sources;
-	}
-
-	public Collection<Integer> getSinkIDs() {
-		return sinks;
-	}
-
-	public Collection<StreamNode> getStreamNodes() {
-		return streamNodes.values();
-	}
-
-	public Set<Tuple2<Integer, StreamOperatorFactory<?>>> getAllOperatorFactory() {
-		Set<Tuple2<Integer, StreamOperatorFactory<?>>> operatorSet = new HashSet<>();
-		for (StreamNode vertex : streamNodes.values()) {
-			operatorSet.add(new Tuple2<>(vertex.getId(), vertex.getOperatorFactory()));
-		}
-		return operatorSet;
-	}
-
-	public String getBrokerID(Integer vertexID) {
-		return vertexIDtoBrokerID.get(vertexID);
-	}
-
-	public long getLoopTimeout(Integer vertexID) {
-		return vertexIDtoLoopTimeout.get(vertexID);
+	Map<Integer, Long> getVertexIDtoLoopTimeout() {
```

Comment:
Why not replacing this method with:

```
boolean isPartOfIteration(int vertexId) {
		return vertexIDtoLoopTimeout.containsKey(vertexId);
	}
```

It seems that this is the way we use it in all other places.